x-pack/legacy/plugins/canvas/public/style/main.scss

```diff
@@ -40,22 +39,22 @@ $canvasElementCardWidth: 210px;
   border-top: $euiBorderThin;
 }
 
-#canvas-app {
+#canvas-app { // sass-lint:disable-line no-ids
   overflow-y: hidden;
 
   .window-error {
     position: absolute;
     bottom: 10px;
     right: 10px;
-    background-color: red;
-    color: white;
+    background-color: $euiColorDanger;
+    color: $euiColorEmptyShade;
     display: block;
     z-index: 2000;
     padding: 10px;
     max-width: 500px;
 
     a {
-      color: white;
+      color: $euiColorEmptyShade;
```

Comment:
`$euiColorGhost` here too... these warning boxes always have a red background (don't flip between light and dark)